lib/dash/dash_parser.js

```diff
@@ -1230,8 +1230,8 @@ shaka.dash.DashParser = class {
 
     let forced = false;
     if (isText) {
-      // See: https://github.com/google/shaka-player/issues/2122
-      forced = roles.includes('forced_subtitle');
+      // See: https://github.com/Dash-Industry-Forum/DASH-IF-IOP/issues/165
+      forced = roles.includes('forced-subtitle');
```

Comment:
Since there will be streams created with the wrong role, I think we should keep the support for the `_` (i'm in that case)

```suggestion
      forced = roles.includes('forced-subtitle') || roles.includes('forced_subtitle');
```

Reply:
Done! Thanks!